android/app/build.gradle

```diff
@@ -181,6 +182,11 @@ task copyThirdPartyLicenses(type: Copy) {
     into "build/generated-assets/3rd-party-licenses"
 }
 
+task copyFonts(type: Copy) {
```

Comment:
Not sure about this. This was solved with a symbolic link. I guess that's the problem with Windows, it does not have symbolic link. But then you're changing code to make it build in a platform we're not totally sure we need to support... So have you considered alternatives to the symbolic link that do not involve changing the code?